infrastructure/validators/operational-tasks/monitor-node.md

```diff
@@ -0,0 +1,371 @@
+- **`console_libraries`** - supporting libraries for the consoles
+
+To set up Prometheus:
+
+1. Copy the binaries to the `/usr/local/bin/` directory:
+  ``` bash
+  sudo cp ./prometheus /usr/local/bin/
+  sudo cp ./promtool /usr/local/bin/
+  ```
+2. Assign ownership of these files to the `prometheus` user:
+  ``` bash
+  sudo chown prometheus:prometheus /usr/local/bin/prometheus
+  sudo chown prometheus:prometheus /usr/local/bin/promtool
+  ```
+3. Copy the `consoles` and `console_libraries` directories to `/etc/prometheus`:
+  ``` bash
+  sudo cp -r ./consoles /etc/prometheus
+  sudo cp -r ./console_libraries /etc/prometheus
+  ```
+4. Assign ownership of these directories to the `prometheus` user:
+  ``` bash
+  sudo chown -R prometheus:prometheus /etc/prometheus/consoles
+  sudo chown -R prometheus:prometheus /etc/prometheus/console_libraries
+  ```
+5. After copying all the necessary files, you can clean up the downloaded Prometheus directory:
+  ```bash
+  cd .. && rm -rf prometheus*
+  ```
+
+## Configure Prometheus
+
+To use Prometheus, you need to configure it by creating a YAML configuration file. This file will define how Prometheus scrapes targets and evaluates rules.
+
+Create a file called `prometheus.yml` in `/etc/prometheus/` as follows:
+```bash
+sudo nano /etc/prometheus/prometheus.yml
+```
+
+This file should include the following key sections:
+
+- **`global`** - defines general parameters, such as scrape intervals
+- **`rule_files`** - points to rule files Prometheus uses to trigger alerts
+- **`scrape_configs`** - lists resources Prometheus will monitor, such as your node
+
+The configuration file should resemble the following:
+
+``` yaml
+--8<-- 'code/infrastructure/validators/operational-tasks/monitor-node/prometheus.yml'
+```
+
+In this example, Prometheus is scraped every 5 seconds to ensure detailed internal metrics. Node metrics with customizable intervals are scraped from port `9615` by default.
+
+You can validate the correctness of your configuration using the `promtool` open-source monitoring system:
+
+```bash
+promtool check config /etc/prometheus/prometheus.yml
+```
+
+Save the configuration file and change the ownership of the file to `prometheus` user:
+
+```bash
+sudo chown prometheus:prometheus /etc/prometheus/prometheus.yml
+```
+
+## Start Prometheus
+
+Use the following command to launch Prometheus with a given configuration, set the storage location for metric data, and enable web console templates and libraries: 
+
+```bash
+sudo -u prometheus /usr/local/bin/prometheus --config.file /etc/prometheus/prometheus.yml --storage.tsdb.path /var/lib/prometheus/ --web.console.templates=/etc/prometheus/consoles --web.console.libraries=/etc/prometheus/console_libraries
+```
+
+If you set the server up properly, you should see messages similar to the following:
+<!--TODO: this should be run live to get fresh terminal output to add to a styled termynal snippet. These are all dated 2021 -->
```

Comment:
@nhussein11    This terminal output is from 2021. We need to spin this up and get fresh output to put into a styled termynal element. If I get clear/complete enough instructions and prereqs, I can try it, but it would probably be easier and faster for you to do it.